develop/parachain-devs/interoperability/transfer-assets-with-xcm.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Transfer Assets with XCM
+description: This tutorial will guide you through the process of transferring assets between parachains using Cross-Chain Message (XCM) in a Polkadot parachain.
+---
+
+# Transfer Assets with XCM
+
+## Introduction
+
+In the [Open Message Passing Channels](TODO:update-path){target=\_blank} tutorial, you saw how to open a two-way communication channel between chains by sending messages to the relay chain. You can use a similar strategy to send messages that allow a local chain to manage an account on a remote chain. In this tutorial, parachain B transfers assets into the sovereign account on the relay chain for parachain A.
+
+The outcome for this tutorial is similar to using the `transfer` function from the balances pallet, except in this case the transfer is initiated by a parachain and demonstrates how the holding register is used when executing the `WithdrawAsset` and `DepositAssetXCM` instructions.
+
+## Prerequisites
+
+Before you begin, verify the following:
+
+- You have the Sudo pallet available for both local parachains to use
+- You have opened the message passing channel to allow communication between parachain B and parachain A
+- You have set up a parachain test network using Zombienet or a local relay chain using the `rococo-local` chain specification. This tutorial uses the following Zombienet configuration file:
```

Comment:
The prerequisite should be something like you have a network with a relay chain and at least two parachains. I would not include the zombienet config used to get this working